requirements.txt

```diff
@@ -4,6 +4,8 @@ Django==1.10
 django_compressor==2.1
 djmail==0.11.0
 futures==3.0.3
+gunicorn
```

Comment:
I would advise that we have the versions of these packages: `gunicorn` and `whitenoise` explicitly stated.